server.js

```diff
@@ -684,6 +684,33 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Coverity Code Advisor On Demand integration
+camp.route(/^\/coverity\/ondemand\/(.+)\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var badgeType = match[1]; // streams or jobs
+  var badgeTypeId = match[2]; // streamId or jobId
```

Comment:
I see. Could we have a static badge in `try.html` to show an example of what it would look like?